### ISSUE TITLE
Failed test 'Runner Usage Message is printed'

### ISSUE THREAD

**Issue by eserte** [bodo-hugo-barwich/Process#48]
On most smoker systems the test suite is failing like this:
```
    #   Failed test 'Runner Usage Message is printed'
    #   at t/test_runner.t line 78.
    # Looks like you failed 1 test of 4.

#   Failed test 'Runner Script Usage'
#   at t/test_runner.t line 79.
# Looks like you failed 1 test of 7.
t/test_runner.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/7 subtests 
```

**Comment by bodo-hugo-barwich** [comment]
Thank you for your feedback. I'm tracking this issue also for some time now.

This is also related to the issues:
[Failing Test Verbose Output](https://github.com/bodo-hugo-barwich/Process/issues/47)
[test_runner.t fails on some Perl Versions](https://github.com/bodo-hugo-barwich/Process/issues/39)

**Comment by bodo-hugo-barwich** [comment]
This error could be fixed with Release `v2.1.10`
[`Process::SubProcess` Release `v2.1.10`](https://github.com/bodo-hugo-barwich/Process/releases/tag/v2.1.10)

Thank you for reporting it!
